### ISSUE TITLE
Sinric Pro temperature sensor refreshing issue

### ISSUE THREAD

**Issue by pedrofnm** [sinricpro/help-docs#9]
Hello, 
I have 3 temperature/humidity sensors in my sinric Pro account  and I'm planning getting 2 more but I'm facing an issue that I can't get througt. The thing is the sensors works great but it supposed to update temp/humidity every minute and it's not happening. Sometimes it updates kinda randomly but other times it doesn't unless I unplug it and plug it back on. What I'm trying to say is that it only works well when I restart the sensor. I'm using the code provided by sinric Pro (only edited ssid, pass, device ID, pin number, etc.) And I'm using an esp 01 s dht11 sensor module. All the sensors are close to the router so I don't know what else might be causing this problem. I even uncommented Enable_debug in the code to see if something changes but nothing.  I'm not sure if the code uses deep sleep or something and it won't update because of that... Anyway I have a picture of my sinric Pro dashboard showing that some sensors stays hours without updating even if I refresh the dashboard. I hope you can help me on this.

Thanks
![Screenshot_20210525-120231](https://user-images.githubusercontent.com/64743076/119490795-283eb480-bd55-11eb-9c09-3e0ba4273959.png)

**Comment by kakopappa** [comment]
Have you tried disabling the sleep?
https://github.com/esp8266/Arduino/issues/460

Also try adding remoteddebug
https://github.com/JoaoLopesF/RemoteDebug

And monitoring the serial output over time

**Comment by pedrofnm** [comment]
Well I think I found what's going on. It only updates in the dashboard if a temp/humidity value is changed. If the values stays the same, it doesn't update in the dashboard since the temp/humidity values don't change much in a closed room. I have tried to point a heater to the sensor and it automatically updated the raised temp. Until this, the sensor was like 1 hour without updating because the temperature/humidity values hadnt changed in that sensor. And I have done this to the other sensors and updated automatically too.

**Comment by pedrofnm** [comment]
I guess I misunderstood how the graph in the dashboard works. I thought it would update every minute the temperature/humidity even if was the same values.

**Comment by kakopappa** [comment]
If you have used the sample sketch, it does not send the same value again and again
https://github.com/sinricpro/esp8266-esp32-sdk/blob/master/examples/temperaturesensor/temperaturesensor.ino#L107

**Comment by pedrofnm** [comment]
Yeah I did used the sample sketch. Next time I should pay more attention to the code :/ 
Thanks for your support
